XcodeServerSDK/Utils/JSON.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 public protocol JSONSerializable {
-    init?(json: NSDictionary)
+    init?(json: NSDictionary) throws
```

Comment:
On a side note I'm noticing some tests complaining due to the lack of `do`, `try`/`catch` blocks on some calls.

Will fix those and then will commit here again